### DISCUSSION TITLE
[bug] email sent to wrong user

### DISCUSSION THREAD

**Discussion by mtdvlpr** [community/community#45307] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi, it seems when I mention user A in a message, user B receives an email about being mentioned. See https://github.com/sircharlo/meeting-media-manager/discussions/674#discussioncomment-4774549. I mentioned stecchio66, but someone else who was not even part of the discussion (or repository even) received an email about being mentioned. It happened twice already.

**Comment by airtower-luna** [comment, marked as answer]
You might want to [send a bug report directly to GitHub support](https://support.github.com/contact/bug-report), so they take a look.

**Reply by mtdvlpr** [reply]
I did and they just responded. Thanks!
